services/ui/serverless.yml

```diff
@@ -216,6 +216,51 @@ resources:
             CachePolicyId: Managed-CachingDisabled
             TargetOriginId: some_target_origin_id
             ViewerProtocolPolicy: allow-all
+    # CloudFrontDistribution:
```

Comment:
Having this in there ready to comment back into place to replace the simplified version may be helpful if we ever need to undo things.